README.md

```diff
@@ -415,7 +415,34 @@ Logger.getInstance().setLogLevel(Logger.LogLevel.Verbose);
   adb logcat > "C:\logmsg\logfile.txt"
  ```
  More examples about adb cmds: https://developer.android.com/tools/debugging/debugging-log.html#startingLogcat
- 
+
+#### Telemetry
```

Comment:
+1 on Wiki over README (even for ADAL).  A snapshot of telemetry may make sense, but if we want to show the schema, more advanced usage (sending it to a service), and more detail overall the Wiki is the right spot.